css/catalog.scss

```diff
@@ -32,3 +32,7 @@
   margin-top: 3%;
   margin-bottom: 3%;
 }
+
+footer.footer {
```

Comment:
@MattiSG : It is in order, but I thought that it was a bad idea to rely on this. It seems I'm wrong, am I ?